pkg/trace/call.go

```diff
@@ -116,10 +116,10 @@ func EndCall(ctx context.Context) {
 			case statuscodes.CategoryServerError:
 				log.Error(ctx, info.Name, info, IDs(ctx), traceEventMarker{})
 			case statuscodes.CategoryOK: // just in case if someone will return non-nil error on success
-				log.Info(ctx, info.Name, info, IDs(ctx), traceEventMarker{})
+				log.Debug(ctx, info.Name, info, IDs(ctx), traceEventMarker{})
 			}
 		} else {
-			log.Info(ctx, info.Name, info, IDs(ctx), traceEventMarker{})
+			log.Debug(ctx, info.Name, info, IDs(ctx), traceEventMarker{})
```

Comment:
This would really mess with us and require a lot of change on our end.

Reply:
Marking as request changes until we can discuss this further.

Reply:
@clevelittlefield-outreach I think based on Slack thread we aren't going to do this.  This prompted discussion and hopefully a push towards design, but this doesn't seem like something Sync will own https://outreach-hq.slack.com/archives/C015GHLM5EK/p1658438757637789